packages/next/server/lib/clearConsole.ts

```diff
@@ -0,0 +1,5 @@
+export default function clearConsole() {
```

Comment:
We don't do console clearing in Next.js as this approach clear the complete screen including all history of the current terminal session.